docs/student-parent-discussion.txt

```diff
@@ -0,0 +1,25 @@
+title Student/Parent Discussion
+
+actor User
+participant UI
+database Data Store
+
+note over User,UI:User (student/parent) logged in
+
+activate User
+activate UI
+User->UI: Select school in shared list
+
+activate Data Store
+UI->Data Store: Request school data
+UI<--Data Store:  Return data
+User<--UI:  Display data
+
+User->UI: Enter comment/discussion interface
+UI->Data Store: Request discussion history
+UI<--Data Store: Return discussion history
+User<--UI: Display discussion history
+User->UI: Leave comment
+deactivate User
+
+UI->Data Store:save comment in DB
```

Comment:
we can also add the last line saying "show comment (posted) to on the interface."

A question to all of us, are we going to send a notification to the child/parent (email/browser-based) saying that you have a new discussion on this particular school so that people can go and check out.

Reply:
A cool feature, but I think it would better in icebox if we want to do it. Currently online display notification alert would suffice